---
bool isRadiale
int32 id
float64[] longitude
float64[] latitude
int32 remainingMissions
